### ISSUE TITLE
Stable Diffusion 2-1 Not Loaded on Colab with Minimal Loading

### ISSUE THREAD

**Issue by maliozer** [huggingface/diffusers#6446]
### Describe the bug

Within this simple loading script:
```python
from diffusers import DiffusionPipeline

pipeline = DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-2-1")
```
The progress is stucked there is no error at all, 
Fetching 13 files: 0% 0/13 [05:54<?, ?it/s]

### Reproduction

Colab: https://colab.research.google.com/drive/1tzys4nVo95JAJgd_0MWp4s9Nm71wxQlZ?usp=sharing

### Logs

```shell
Fetching 13 files: 0%
0/13 [05:54<?, ?it/s]
tokenizer/special_tokens_map.json: 100%
460/460 [00:00<00:00, 9.94kB/s]
tokenizer/tokenizer_config.json: 100%
824/824 [00:00<00:00, 50.3kB/s]
```

### System Info

Colab
- `diffusers` version: 0.25.0
- Platform: Linux-6.1.58+-x86_64-with-glibc2.35
- Python version: 3.10.12
- PyTorch version (GPU?): 2.1.0+cu121 (False)
- Huggingface_hub version: 0.20.1
- Transformers version: 4.35.2
- Accelerate version: not installed
- xFormers version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

### Who can help?

@yiyixuxu @DN6 @sayakpaul @patrickvonplaten

**Comment by sayakpaul** [comment]
That is because of bandwidth problems. It's not a library issue. I was able to successfully load it on Colab just a short while ago.

**Comment by DN6** [comment]
The issue has also been raised on the Colab repo
https://github.com/googlecolab/colabtools/issues/4272

**Comment by pranauv1** [comment]
It's very weird, but running it on the `.py` file seems to be working.

**Download the model through Python**
```python
from diffusers import DiffusionPipeline
import torch

DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, use_safetensors=True, variant="fp16")
```
Save the above as a py file and run it on Colab, now the model will be cached!

**Call the pipeline**

`pipeline = DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, use_safetensors=True, variant="fp16")`

_Now this won't attempt to download from HF since the model is cached locally!_

**Comment by ritwikraha** [comment]
I don't know if this is practical for everyone but running it on `kaggle-notebooks` seems to be working

https://www.kaggle.com/ritzraha/diffusion-template

So it is highly probable that this is a colab issue, FYI I am still having the same problem (as documented above) on colab

**Comment by imanphung** [comment]
`pipe = StableDiffusionControlNetPipeline.from_pretrained(
    "SG161222/Realistic_Vision_V5.1_noVAE",
    controlnet=controlnet,
    torch_dtype=torch.float16,
    scheduler=noise_scheduler,
    vae=vae,
    token="",
).to("cuda")`

I added token inside then it works fine!

**Comment by maliozer** [comment]
It may await to read hf_token from colab but never access

**Comment by KevinJBoyer** [comment]
@imanphung thank you! Confirming that adding `token=""` to the `.from_pretrained` call worked for me too.

**Comment by maliozer** [comment]
> @imanphung thank you! Confirming that adding `token=""` to the `.from_pretrained` call worked for me too.

This is not worked for me, still need a valid solution

**Comment by math-rb** [comment]
> It's very weird, but running it on a `.py` file seems to be working.
> 
> **Download the model through a Python file**
> 
> ```python
> from diffusers import DiffusionPipeline
> import torch
> 
> DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, use_safetensors=True, variant="fp16")
> ```
> 
> Save the above as a `.py` file and run it on Colab, the model will be cached!
> 
> **Now call the pipeline** _This won't attempt to download from HF since the model is already cached locally!_
> 
> ```python
> pipeline = DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, use_safetensors=True, variant="fp16")
> ```

Worked here.

**Comment by sayakpaul** [comment]
We're on top of it: https://github.com/huggingface/huggingface_hub/pull/1953. Cc: @Wauplin

**Comment by Wauplin** [comment]
Thanks everyone for reporting and sorry for the inconvenience. I have been able to reproduce the issue and created a PR (https://github.com/huggingface/huggingface_hub/pull/1953) to fix it. The plan is to make a hot-fix release once it's merged and make it the default in google colab. In the meantime, a temporary solution is to install a different version of `huggingface_hub` before executing any code. More details in https://github.com/huggingface/huggingface_hub/pull/1953#issuecomment-1878364894.

**Comment by Wauplin** [comment]
**Update:** `huggingface_hub 0.20.2` is now the default version on Google Colab, meaning this issue should be fixed for everyone without having to upgrade (see https://github.com/googlecolab/colabtools/issues/4272#issuecomment-1887683158).
